tasks/main.yml

```diff
@@ -5,16 +5,26 @@
 - include: setup-Debian.yml
   when: ansible_os_family == 'Debian'
 
+- name: Define nodejs_install_npm_user.
+  set_fact:
+    nodejs_install_npm_user: "{{ ansible_ssh_user }}"
```

Comment:
Could we just set this as the default in `defaults/main.yml` and then remove this task entirely? Then there will always be a value, and it would default to `ansible_ssh_user`.